### PR TITLE
Increased recycle center mundane loot tenfold

### PR DESCRIPTION
**Pull request by RarkGrames**
```SUMMARY: Content "Increased recycle center mundane loot tenfold"```

#### Purpose of change
Increasing the amount of mundane items (glass jars, cardboard boxes, glass bottles, nails etc.) in recycle centers.
Contributes to #34432.

#### Describe the solution
- Increasing the item placements in mapgen tenfold
- Dropping the rate of useful items like books, tools, storage batteries, etc.
- Increasing the storage capacity of the recycle bin (from 20L to 200L)

#### Testing
Spawned all three variants. Looting ended with 2-3 dozen glass jars, same amount of glass bottles, a couple dozen cardboard boxes. Found roughly the same amount of valuable loot as before.

### CODE REVIEW

**Comment by mkikt4743** [comment]
maybe 3x but 10x seems a little much, I don't think recycling is a big thing in new england

**Comment by RarkGrames** [comment]
> 
> 
> maybe 3x but 10x seems a little much, I don't think recycling is a big thing in new england

Around 70% of the bins were empty before. Now you have 5-15 items in them. I think that is still around the lower level.

File: data/json/itemgroups/locations.json [hunk 1]
```diff
@@ -588,19 +588,19 @@
...
-      [ "box_medium", 35 ],
-      [ "box_large", 15 ],
-      [ "one_year_old_newspaper", 45 ],
-      [ "months_old_newspaper", 50 ],
-      [ "weeks_old_newspaper", 50 ],
-      [ "one_year_old_newspaper", 45 ],
-      [ "years_old_newspaper", 40 ],
-      [ "many_years_old_newspaper", 40 ]
+      [ "flyer", 120 ],
+      { "item": "paper", "prob": 500, "charges-min": 1, "charges-max": 15 },
```
Comment (I-am-Erk) [inline data/json/itemgroups/locations.json]:
```suggestion
      { "item": "paper", "prob": 500, "charges-min": 1 },
```
No reason to cap the amount of paper, imo

File: data/json/itemgroups/locations.json [hunk 2]
```diff
@@ -734,23 +734,23 @@
...
-      { "item": "nail", "prob": 65, "charges-min": 1, "charges-max": 15 },
-      [ "well_pump", 15 ],
+      [ "pastaextruder", 75 ],
+      { "item": "nail", "prob": 300, "charges-min": 1, "charges-max": 15 },
+      [ "well_pump", 50 ],
       [ "manhole_cover", 20 ],
-      [ "hairpin", 40 ],
-      [ "pan", 30 ],
+      [ "hairpin", 120 ],
+      [ "pan", 90 ],
```
Comment (I-am-Erk) [inline data/json/itemgroups/locations.json]: I've added a ton of kitchens pots and pans that could be dumped into this group too

Comment (RarkGrames) [reply data/json/itemgroups/locations.json]: I'll look around what can I throw in.

**Comment by RarkGrames** [comment]
Added some more items.

**Comment by Maleclypse** [comment]
> maybe 3x but 10x seems a little much, I don't think recycling is a big thing in new england

New Hampshire is the only state in New England without some form of bottle tax that is refunded by recycling.  Recycling machines for bottles and cans are at every grocery store.

**Comment by ashGlaw** [comment]
There are lots of bottles and cans dropped off at the transfer station every week. 

Depending on what day of the week the cataclysm happened, either there'd be a ton of recyclable loot at the transfer station, or there'd be a bunch of recyclable loot at every home, waiting to be brought out/picked up. Recycling is big in Massachusetts. I've had weekly recycling pickup at every place I've lived in Mass, even when I lived way out in the sticks.

**Pull request merged**